### PR TITLE
Fix anaconda validation issue

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
JIRA: https://issues.redhat.com/browse/RHOAIENG-1899

## Description

Our backend is reading the `CronJob` of the `batch/v1beta1` group while that's deprecated and replaced by the `batch/v1` group. Changed the backend to read from the correct group to fetch the resources.
Fixed a small layout issue with the upgrading of PF5 in the enable modal:

Screenshot 2024-02-09 at 10 13 46 AM

## How Has This Been Tested?

1. Find a cluster installed OpenShift AI
2. Enable anaconda
3. Check it works

## Test Impact

N/A

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: backend/src/routes/api/validate-isv/validateISV.ts [hunk 1]
```diff
@@ -102,12 +102,18 @@ export const runValidation = async (
...
+    .then((res) => res.body as CronJobKind)
     .catch(() => {
       fastify.log.error(`validation cronjob does not exist`);
     });
+  // const cronJob = await batchV1beta1Api
+  //   .readNamespacedCronJob(cronjobName, namespace)
+  //   .then((res) => res.body)
+  //   .catch(() => {
+  //     fastify.log.error(`validation cronjob does not exist`);
+  //   });
```
Comment (andrewballantyne) [inline backend/src/routes/api/validate-isv/validateISV.ts:116]: Why didn't you delete the commented code?

**Review by andrewballantyne — CHANGES_REQUESTED** [review]
Looks good on paper -- will need to have someone test this -- please ask the team

**Comment by christianvogt** [comment]
Changes look good. Haven't ran the PR though.

**Pull request merged**